source/incident-collaboration/playbook-planning.rst

```diff
@@ -1,47 +1,67 @@
+6. Choose **Save**.
   
-Setting preferences
-~~~~~~~~~~~~~~~~~~~
+Configuring templates
+---------------------
+
+Templates are standardized sets of content that are used for communicating reminders and updates.
+
+1. Go to **Main Menu > Incident Collaboration**.
+2. Select **Playbooks**.
+3. Select the **Templates** tab.
 
-1. Select the **Preferences** tab.
-2. Within the **Preferences** tab, customize how status updates are communicated:
   * Optionally configure a broadcast channel to which status updates will be copied. If you are not a member of the configured broadcast channel, **Unknown Channel** is displayed instead of the channel name.
   * Optionally configure the default reminder timer used to prompt for regular updates. The reminder timer may be changed when a status update is written.
   * Optionally configure a template to use for the first status update. Subsequent status updates will start with the text of the most recent update.
   
-Setting automation
-~~~~~~~~~~~~~~~~~~
-
-1. Select the **Automation** tab.
-2. Within the **Automation** tab, customize automatically triggered tasks:
-  * Optionally enable the **Invite members** toggle and select a set of members. This set of members are automatically invited to the incident channel when the incident starts.
-  * Optionally enable the **Assign commander** toggle and select a member. This member is automatically assigned as commander of the incident when the incident starts.
-  * Optionally enable the **Announce it in another channel** toggle and select a channel. When the incident is started, an announcement is made in the selected channel.
-  * Optionally enable the **Send a webhook** toggle and enter the webhook you want to use.
-3. On the right-hand side of the screen, configure the permissions:
-  * Decide whether the automatically-created incident channel should be Public or Private within the team.
-  * Share this playbook with other members of the team to allow them to use the playbook to start an incident, as well as edit the playbook.
+Defining actions
+----------------
+
+You can customize actions associated with your playbook by setting up keyword triggers, automate member invites, and add a welcome message for new members.
+
+Select the **Actions** tab.
+
+  * **Keywords**: Enable the toggle, and enter comma-separated keywords that, when included in a message, will prompt the posting user to run the playbook.
+  * **Invite members**: Enable the toggle, and select members who will automatically be invited to the channel when the playbook run starts.
+  * **Assign the owner role**: Enable the toggle and select a member. This member is automatically assigned as the owner of the playbook run.
+  * **Announce in another channel**: Enable the toggle and select a channel. When the playbook run is started, an announcement is made in the selected channel.
+  * **Send outgoing webhook**: Enable the toggle and enter the webhook you want to use for status updates.
```

Comment:
We also have **Send outgoing webhook** when the incident status is updated and **Send a welcome message** when a new member joins the channel.
![image](https://user-images.githubusercontent.com/3924815/125492402-04faedcd-8720-43fc-ab5c-e11f746e8226.png)